### ISSUE TITLE
Make EnkiApp customisable

### ISSUE THREAD

**Issue by schernichkin** [PROTEINE-INSAIDERS/enki#18]
User should be able to extend EnkiApp to accept custom arguments in existent commands as well as custom commands.

PoC use-case: pass parameter which overrides table name for one particular table.

**Comment by schernichkin** [comment]
Some job already done in 44a1ebf5fa5f4fcbd488518d9ccf2b148e658fc5 by introducing trait EnkiMain.

**Comment by schernichkin** [comment]
PoC may heavily depend on #25
